diff --git a/src/Command/Config/ImportCommand.php b/src/Command/Config/ImportCommand.php
index 1d096340..c068536e 100644
--- a/src/Command/Config/ImportCommand.php
+++ b/src/Command/Config/ImportCommand.php
@@ -88,15 +88,11 @@ class ImportCommand extends Command
         $skipUuid = $input->getOption('skip-uuid');
 
         if ($directory) {
-            $configSyncDir = $directory;
+            $source_storage = new FileStorage($directory);
         } else {
-            $configSyncDir = config_get_config_directory(
-                CONFIG_SYNC_DIRECTORY
-            );
+            $source_storage = \Drupal::service('config.storage.sync');
         }
 
-        $source_storage = new FileStorage($configSyncDir);
-
         $storageComparer = '\Drupal\Core\Config\StorageComparer';
         if ($skipUuid) {
             $storageComparer = '\Drupal\Console\Override\StorageComparer';
